app/renderer/components/preferences/syncTab.js

```diff
@@ -56,7 +65,7 @@ class SyncTab extends ImmutableComponent {
     }
     return this.props.syncQRVisible
       ? <div>
-        <div><Button l10nId='syncHideQR' className='whiteButton syncToggleButton' onClick={this.props.hideQR} /></div>
+        <div><Button l10nId='syncHideQR' className='whiteButton wideButton syncToggleButton' onClick={this.props.hideQR} /></div>
```

Comment:
Do we need this one? the min-width of the buttons has been set globally with https://github.com/brave/browser-laptop/pull/6387